cmd/configure.ac

```diff
@@ -178,5 +178,22 @@ AC_PATH_PROG([HAVE_VALGRIND],[valgrind])
 AM_CONDITIONAL([HAVE_VALGRIND], [test "x${HAVE_VALGRIND}" != "x"])
 AS_IF([test "x$HAVE_VALGRIND" = "x"], [AC_MSG_WARN(["cannot find the valgrind tool, will not run unit tests through valgrind"])])
 
+# This knob controls if builds snap-confine are done in a pretty unusual way.
+# The goal is to link dynamically to libc and anything that talks to remote
+# things that we cannot control and that have a volatile protocol, specifically
+# this is udev.
+#
```

Comment:
I would turn around the phrasing here:

"""
This option controls whether snap-confine should be built largely statically.
The goal is to link dependencies statically except for libc and anything that talks to lower system components we cannot control  via a volatile protocol, this is udev.

This is done so that ...
"""

I wonder also if we should  s/partially/largely/  in the option and var